.gitban/roadmap/roadmap-usage.md

```
@@ -0,0 +1,161 @@
+```python
+page = read_roadmap(path="m1", limit=50)        # first 50 children
+while page["pagination"]["has_more"]:
+    page = read_roadmap(path="m1", limit=50,
+                        offset=page["pagination"]["next_offset"])
+```
+
+## Error Recovery
+
+When a tool returns an error, it includes the available siblings to help you navigate:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)